supertokens_python/recipe/thirdparty/__init__.py

```diff
@@ -39,6 +43,11 @@
 
 def init(sign_in_and_up_feature: SignInAndUpFeature,
          email_verification_feature: Union[InputEmailVerificationConfig, None] = None,
-         override: Union[InputOverrideConfig, None] = None) -> Callable[[AppInfo], RecipeModule]:
+         override: Union[InputOverrideConfig, None] = None,
+         email_delivery: Union[EmailDeliveryConfig[TypeThirdPartyEmailDeliveryInput], None] = None
+         ) -> Callable[[AppInfo], RecipeModule]:
     return ThirdPartyRecipe.init(
-        sign_in_and_up_feature, email_verification_feature, override)
+        sign_in_and_up_feature, email_verification_feature, override, email_delivery)
+
+# async def send_email(input: TypeThirdPartyEmailDeliveryInput, user_context: Dict[str, Any]):
+#     return await ThirdPartyRecipe.get_instance().email_delivery_ingredient.send_email(input, user_context)
```

Comment:
Why this comment?

Reply:
Fixed. Added send_email in syncio/asyncio.